pkg/cmd/workspace/util/repository_wizard.go

```diff
@@ -134,7 +143,12 @@ func getRepositoryFromWizard(config RepositoryWizardConfig) (*apiclient.GitRepos
 			disablePagination = true
 		} else {
 			// Check if we have reached the end of the list
-			disablePagination = int32(curPageItemsNum) < perPage
+			// For few providers, we manually append "personal" namespace info on the first page
+			if isGitProviderAppendsPersonalNamespace(providerId) {
```

Comment:
Shouldn't we also check that it's the first page?

Reply:
fixed, everythings wrong with my brain today